src/commenter.js

```diff
@@ -11,15 +11,15 @@ const getDedupeComment = (content) => `<!-- post-github-comment-from-travis :: $
  * Get the unique token for the comment.
  *
  * @param {string} repository The repository, in slug form.
- * @param {string|number} pullRequest The pull request number on GitHub.
+ * @param {string|number} issue_number The issue number on GitHub.
  * @param {string} purpose The purpose of the comment.
  */
-function getUniqueToken({ repository, pullRequest, purpose }) {
+function getUniqueToken({ repository, issue_number, purpose }) {
```

Comment:
Any reason we need snake case here? Asking since it's not a positional parameter

Reply:
Because that's what `octokit` uses and it's just easier to be consistent in this case - it's what we receive from `@mixmaxhq/travis-utils`' `getOptions` call.

Reply:
NVM assuming it's to match up with the octokit API